### ISSUE TITLE
Accessibility: When zoomed to 400%, few options text is not completely visible in Tag Picker

### ISSUE THREAD

**Issue by v-sagsah** [microsoft/fluentui#13289]
**Browser Details: ​**
OS: Windows 10 (Version 1909, Build: 18363.778)
Microsoft Edge Dev Version 84.0.502.0 (Official build) dev (64-bit)
Chrome Version 81.0.4044.138 (Official Build) (64-bit)

**Issue:**
When zoomed to 400%, few options text is not completely visible in the Tag Picker.

Issues is reproducible in the below code-pen link :
~~https://codepen.io/sagsah/full/RwWxxpw~~ *new link: https://codepen.io/ecraig12345/full/LYxammw*

![image](https://user-images.githubusercontent.com/64409491/82677975-76166d80-9c66-11ea-96c8-012924ef72f9.png)

**Expected Result: ​**
Every text and control should be visible and accessible when zoomed to 400%.

**User Impact: ​**
Low vision users may find it difficult to understand the complete value of the option while navigating through the auto suggestion dropdown.

**Comment by aneeshack4** [comment]
@v-sagsah We’re currently focusing on improvements to performance and coherence, as you can see in our pinned issue on the repo detailing our v8 plans. So while we don’t plan to address this right now, we will take it into consideration when we do future work on TagPicker. Thanks for reaching out to us!

**Comment by ecraig12345** [comment]
Looks like this is still valid. Here's an updated codepen: https://codepen.io/ecraig12345/full/LYxammw
